docker/scripts/entrypoint.sh

```diff
@@ -68,6 +123,7 @@ echo "ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE}"
 
 ZKV_CONF_BASE_PATH=${ZKV_CONF_BASE_PATH:-}
 ZKV_CONF_CHAIN=${ZKV_CONF_CHAIN:-}
+ZKV_SPEC_FILE_URL="${ZKV_SPEC_FILE_URL:-}"
```

Comment:
This name is diff to what I have used in Horizen repo. I think we should be on the same page either this or the other one. I went with `RELAYCHAIN_SPEC_FILE_URL` since it is easier to understand

Reply:
I think it is better to stick with "ZKV" parameter ; let's either change it in horizen repo (ZKV instead of RELAYCHAIN)